extra/exports

```diff
@@ -182,6 +182,8 @@ fiber_set_cancellable
 fiber_set_joinable
```

Comment:
* Please, ensure the CI is green in your PR. Currently you fail linter checks.
* Please, adhere to our commit message standards. I.e. in your case at least the title length should be <= 50 and the subsystem name should be just `fiber:` not `ffi(fiber)`.
* You need to add a [docbot request](https://github.com/tarantool/docbot#docbot---tarantool-documentation-pipeline-bot) (also see `git log` for examples) with a description of the newly exported functions and a short changelog.

Reply:
done

Reply:
Please, don't resolve my comments in the future - I use them to double-check if they are fixed and then resolve them myself.